tasks.go

```diff
@@ -13,8 +13,8 @@ type Task func() error
 
 // RunAllTasks runs all tasks known to the dump tool using concurrent workers.
 // Dump output goes to path.
-func RunAllTasks(path string, workers int) {
-	tasks := GetAllTasks(path)
+func RunAllTasks(runner Runner, path string, workers int) {
```

Comment:
Well observed.

This is temporary, we can't remove it yet because definitions, nagios, and analysis checks still depend on it.
It gets removed with the future commits that refactor the other parts.